pages/support/[category]/[subcategory].tsx

```diff
@@ -1,21 +1,59 @@
 import BottomNav from '../../../components/BottomNav'
-import styles from '../../../styles/Page.module.css'
+import pageStyles from '../../../styles/Page.module.css'
+import styles from '../../../styles/SupportListing.module.css'
 import TopNav from '../../../components/TopNav'
+import Card from '../../../components/support/Card'
 import { useRouter } from 'next/router'
 
 export default function SupportListing() {
+  const support_listing = [
```

Comment:
It would be good if we find a way to mock this coming from the backend. This is ok for now, but let's make sure others don't think we should fill in all the data this way. :D